py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -42,6 +42,27 @@
 
 HWW_CMD = 0x80 + 0x40 + 0x01
 
+
+class HwwRequestCode:
+    # New request.
+    HWW_REQ_NEW = b"\x00"
```

Comment:
prefixes not needed? unlike C those are luckily not in the global namespace :)

Reply:
Fixed 2756844

Reply:
+ 136a47f